tests/data/sample-erratum.json

```diff
@@ -701,6 +701,24 @@
     "pulp_user_metadata": {
         "content_types": [
             "module"
+        ],
+        "container_list": [
+            {
+              "e2e/test-operator-container": {
+                "digest": "sha256:2321a7d13d9fa53f05437663cf2dc217d15f3cda4b67076c941b10f0491cf9d7",
+                "images": {
+                  "docker-image-sha256:06d1c5e4fa6a5d1ff868388f3feadf193d04128b62d1181e37fe4ab8ecda27e1.s390x.tar.gz": {
+                    "digest": null
```

Comment:
What's the digest expected to be in practice?  Can we please get at least one case where it's non-null?